README.md

```diff
@@ -47,6 +47,10 @@ to run `gostatsd` with a graphite backend and a grafana dashboard.
 While not generally tested on Windows, it should work.  Maximum throughput is likely to be better on
 a linux system, however.
 
+The server listens for UDP packets by default. Unix Domain Sockets (UDS) can be used specifying a file path instead 
+of `address:port` with the `metrics-addr` configuration option. This only works on linux and will ignore `conn-per-reader` 
+configuration option.
```

Comment:
Somewhere the docs should mention the socket mode is SOCK_DGRAM not SOCK_STREAM

Reply:
I've added it to this paragraph in https://github.com/atlassian/gostatsd/pull/472/commits/11d015b2bd6251deb18b8db3ae611374181465aa. I wasn't sure if it was too much info about the socket in this section and I thought about adding another section just for this. Let me know what do you think and I can quickly change it.